synapse/tests/test_tools_migrate200.py

```diff
@@ -1,1493 +0,0 @@
-import os
```

Comment:
We also need to remove the associated documentation for this tool.

Reply:
so there is additional clean up for doing that and ensuring that we have links back to the the v2.8.0 docs in place